pkg/config/marin3r.go

```diff
@@ -1,7 +1,7 @@
 package config
 
 import (
-	marin3rv1alpha1 "github.com/3scale/marin3r/apis/marin3r/v1alpha1"
```

Comment:
the adding of the `-ops` into these paths, what other affects does this mean? I am thinking does this change paths to quay, does this affect the EWS? These are open ended questions and I am not sure what answers I am looking for. Any other information you can give would be useful.

Reply:
I dont think it has much affect unless these paths are hardcoded elsewhere as this is a dependency change :thinking: 

I believe this change is due to the repo is now https://github.com/3scale-ops/marin3r but previous it was https://github.com/3scale/marin3r
* https://github.com/3scale-ops/marin3r/blob/v0.7.0/bundle/manifests/marin3r.clusterserviceversion.yaml#L139
* https://github.com/3scale-ops/marin3r/blob/v0.8.0/bundle/manifests/marin3r.clusterserviceversion.yaml#L139

In terms of quay, they are still using the same quay repo https://quay.io/repository/3scale/marin3r?tag=latest&tab=tags so there doesn't look to be changes needed there. 

In terms of EWS, in not sure is marin3r being watched it and am not familiar with the system itselft to comment whether this affects it :thinking: :shrug:

Reply:
If this is the same quay.io repo I want to check one of the pipelines that use the products.yaml file. I still your changes are correct but the pipeline may need adjusting.

Reply:
I had a look into what I felt might have needed some follow on work and it doesn't. We are all good.